src/arutils.py

```diff
@@ -701,7 +724,7 @@ def robust_polyfit(xarray, yarray, order, weights=None, maxone=True, sigma=3.0,
         yrng = func_val(ct, xarray, function, minv=minv, maxv=maxv)
         sigmed = 1.4826*np.median(np.abs(yfit-yrng[w]))
         if debug:
```

Comment:
Do we still need the debug keyword? (recall, all instances in PyCharm of this function can be found with cmd+left click).